<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tareas7</class>
 <widget class="QDialog" name="Tareas7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Diferencia1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Diferencia 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Diferencia2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Diferencia 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Intersepcion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Interseptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Limpiar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valore">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Numeros">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConjuntoA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Conjunto A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConjuntoB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Conjunto B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConjuntoR">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Conjunto R</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListaA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ListaB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ListaR">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>71</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
